src/main/oauth2/oauth2.ts

```
@@ -77,7 +77,7 @@ export async function authenticate(): Promise<TokenResponse> {
         openInBrowser(url);
         const callbackUrl = await redirectHandler.waitForCallback();
         log.debug(`Received callback URL: ${callbackUrl}`);
-        const code = callbackUrl.searchParams.get("code");
+        const code = callbackUrl.searchParams.get("code") || "";
```

throw an error if we have no code instead of defining a default value